pkg/crc/preflight/preflight_darwin.go

```diff
@@ -109,6 +109,13 @@ var daemonLaunchdChecks = []Check{
 		cleanupDescription: "Unloading and removing the daemon plist file",
 		cleanup:            removeDaemonPlistFile,
 	},
+	{
+		configKeySuffix:  "check-daemon-is-serving",
+		checkDescription: "Checking if crc daemon is serving the requests",
+		check:            checkIfDaemonIsServing,
+		fixDescription:   "crc daemon is not started (try running manually `crc daemon` command in different terminal)",
```

Comment:
We should not instruct people to run `crc daemon` manually when we are in the process of getting rid of this requirement, especially with all the issues we have when the daemon was already started manually.

Reply:
I can remove that part, my intention was here to get debug info if something really blocked the `crc daemon` which we are not getting with preflight checks.

Reply:
https://github.com/code-ready/crc/pull/3150#discussion_r858899083 same